.github/workflows/main.yml

```diff
@@ -4,7 +4,65 @@ on:
     branches:
     - 'master'
 jobs:
+  list-pkgbuilds:
+    runs-on: ubuntu-latest
+    outputs:
+      aur-pkgs: ${{ steps.set-aur-pkgs.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v3
+      - id: set-aur-pkgs
+        run: source ./manifest ; echo "matrix=$(echo ${AUR_PACKAGES} | jq -R -s -c 'split(" ")[:-1]')" >> $GITHUB_OUTPUT
+        shell: bash
+
+  build-environment:
+    name: Build environment
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-buildx-action@v2
+      - name: build builder image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          tags: image-builder:latest
```

Comment:
Maybe add the commit hash or the chimeraos version? This could be very useful for reproducible builds.